.travis.yml

```diff
@@ -57,7 +57,10 @@ before_script:
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3
 script:
+  - flake8 setup.py
```

Comment:
ok, I see the problem. You want to skip flake traversing the doc directory. I would restore it as is, and use the .flake8 configuration file, like this 

```
 $ more .flake8 
[flake8]
exclude = build/*,venv/*,doc/source/*,tests/utils.py,selenium_tests/*
```